portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -40,9 +44,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     // Create an Entity to store the comment.
     Entity commentEntity = new Entity("Comment");
-    commentEntity.setProperty("commentText", newComment);
+    commentEntity.setProperty(COMMENT_PARAM, newComment);
     commentEntity.setProperty("timestamp", timestamp);
```

Comment:
You shouldn't have any strings like this in your code. Always create a constant for them. See an example below.

https://screenshot.googleplex.com/4BnMVgYvOqq